osquery/tables/system/authorized_keys.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <string>
+#include <vector>
+
+#include <pwd.h>
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+
+namespace osquery {
+namespace tables {
+
+const std::vector<std::string> kSSHAuthorizedkeys = {
+    ".ssh/authorized_keys",
```

Comment:
I like just searching for `authorized_keys`. To be pedantic, one could parse `/etc/ssh/sshd_config` for `AuthorizedKeysFile FILEPATH` but the daemon config is sometimes `o600` so it doesn't matter.